cli/src/config/templates.toml

```diff
@@ -243,6 +238,20 @@ if(commit.hidden(),
 )
 '''
 
+'format_commit_header(commit)' = '''
+separate(" ",
+        format_short_change_id_with_hidden_and_divergent_info(commit),
+        format_short_signature(author),
+        format_timestamp(committer.timestamp()),
+        bookmarks,
+        tags,
+        working_copies,
+        if(git_head, label("git_head", "git_head()")),
+        format_short_commit_id(commit.commit_id()),
+        if(conflict, label("conflict", "conflict")),
```

Comment:
nit: could you update keywords "author", etc. to method calls `commit.author()`, etc.?